bindings/python/manifold3d.cpp

```diff
@@ -460,7 +482,12 @@ NB_MODULE(manifold3d, m) {
           "Gets the manifold bounding box as a tuple "
           "(xmin, ymin, zmin, xmax, ymax, zmax).")
       .def_static(
-          "smooth", [](const Mesh &mesh) { return Manifold::Smooth(mesh); },
+          "smooth",
+          [](const MeshGL &mesh,
+             const std::vector<Smoothness> &sharpenedEdges = {}) {
```

Comment:
Python can't take in the `Smoothness` struct directly, but it can be fixed by replacing it with an `std::tuple<int, float>`:
```cpp
          [](const MeshGL &mesh,
             const std::vector<std::tuple<int, float>> &sharpenedEdges = {}) {
            std::vector<Smoothness> vec(sharpenedEdges.size());
            for (int i = 0; i < sharpenedEdges.size(); i++) {
              vec[i] = {std::get<0>(sharpenedEdges[i]),
                        std::get<1>(sharpenedEdges[i])};
            }
            return Manifold::Smooth(mesh, vec);
          },
```